task03/task03/Program.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace task03
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            Console.WriteLine("Введите трёхзначное число: ");
+
+            int i = Convert.ToInt32(Console.ReadLine());
```

Comment:
Не стоит использовать методы класса Convert без особых причин. Лучше так
```suggestion
            int i = int.Parse(Console.ReadLine());
```